modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -807,9 +807,9 @@ pub enum SendPaymentError {
     InvoiceMissingAmount,
     #[error("The invoice has expired")]
     InvoiceExpired,
-    #[error("A previous payment for the same invoice is still pending: {0}")]
+    #[error("A previous payment for the same invoice is still pending: {}", .0.fmt_full())]
     PendingPreviousPayment(OperationId),
-    #[error("A previous payment for the same invoice was successful: {0}")]
+    #[error("A previous payment for the same invoice was successful: {}", .0.fmt_full())]
     SuccessfulPreviousPayment(OperationId),
```

Comment:
why full here, but ...(see fqsjld)

Reply:
This one looked like an error that could cross API-level boundary, while the other one like just some random ad hoc anyhow thing being yeeted on error.

No strong feelings. Can change the other one to long.